File: csharp/unittest-practice/src/main/TortillaHarina.cs
namespace unittestpractice.main
{
    internal class TortillaHarina
    {
        private bool _toasted;
        private int _temperature;
        private const int Toasting = 10;

        public bool IsToasted()
        {
            return _toasted;
        }

        public int GetToastTemperature()
        {
            return Toasting;
        }

        public int GetCurrentTemperature()
        {
            return _temperature;
        }
        public void SetCurrentTemperature(int temp)
        {
            _temperature = temp;
        }
        public void Toast(bool toast)
        {
            _toasted = toast;
        }


    }
}

File: python/src/main/TortillaHarina.py
class TortillaHarina:
    def __init__(self, toasted=None, temperature=None):
        self.toasting = 10
        self.toasted = toasted
        self.temperature = temperature

    def isToasted(self):
        return self.toasted

    def getToastTemperature(self):
        return self.toasting

    def getCurrentTemperature(self):
        return self.temperature

    def setCurrentTemperature(self, temp):
        self.temperature = temp

    def toast(self, toasted):
        self.toasted = toasted
